src/home/Benefit.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 import './Benefit.css'
-
+import  Benefit1 from "../assets/images/benefit1.png"
+import  Benefit2 from "../assets/images/benefit2.png"
 
 const Benefit = () => {
     return (
         <div className="aboutContainer">
```

Comment:
Maybe we should also rename this className to just "container" to be consistent with src/home/About.jsx line 6 above?

Reply:
Done